covid_model/model_fit.py

```diff
@@ -43,21 +45,22 @@ def func(trange, *test_tc):
                 f=func
                 , xdata=model.trange
                 , ydata=self.actual_hosp[:len(model.trange)]
-                , p0=model.tc[-look_back:]
+                , p0=model.tc[-(look_back-1):] + [self.tc_0]
```

Comment:
shifted over initial guess by one. Should this actually go back `increment_size`?

Reply:
thinking `p0=model.tc[-(look_back-increment_size):] + [self.tc_0]*increment_size`